CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# 80.0.0
+
+* BREAKING: Remove support for import endpoint - this was a temporary endpoint. 
```

Comment:
I'd maybe add a note here to say that no apps were using it, just to make it clearer that this update should be good for all apps